fairscale/optim/grad_scaler.py

```diff
@@ -30,9 +30,10 @@ class ShardedGradScaler(TorchGradScaler):
     documentation https://pytorch.org/docs/stable/amp.html#torch.cuda.amp.GradScaler
     """
 
-    def __init__(self) -> None:
+    def __init__(self, process_group: Any = dist.group.WORLD) -> None:
```

Comment:
this keeps the interface backwards compatible, but makes it possible to select which process group to talk to